requirements-app.txt

```diff
@@ -1,64 +1,51 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
-
-apig-wsgi==2.13.0
+apig-wsgi==2.15.0
 boto==2.49.0
-cffi==1.14.5
-celery[sqs]==5.0.5
+cffi==1.15.1
+celery[sqs]==5.2.7
 docopt==0.6.2
 environs==9.3.2 # pyup: <9.3.3 # marshmallow v3 throws errors
 fido2==0.9.1
-Flask-Bcrypt==0.7.1
+Flask-Bcrypt==1.0.1
 flask-marshmallow==0.14.0
 Flask-Migrate==2.7.0
 git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262#egg=Flask-SQLAlchemy==2.3.2.dev20190108
-Flask==2.0.3
+Flask==2.2.2
 click-datetime==0.2
 eventlet==0.30.2 # currently 0.31.0+ breaks gunicorn. Test the docker image if upgrading!
 gunicorn==20.1.0
 iso8601==0.1.14
-idna==2.8 # pinned to align with test moto dependency requirements
+idna==2.10 # pinned to align with test moto dependency requirements (for <=2.9)
 jsonschema==3.2.0
-marshmallow-sqlalchemy==0.23.1 # pyup: <0.24.0 # marshmallow v3 throws errors
-marshmallow==2.21.0 # pyup: <3 # v3 throws errors
+marshmallow-sqlalchemy==0.23.1
+marshmallow==2.21.0 
 python-magic==0.4.22
-psycopg2-binary==2.8.6
-PyJWT==2.1.0
-pytz==2021.1
+psycopg2-binary==2.9.5
+PyJWT==2.6.0
+pytz==2021.3
 PyYAML==5.4.1
 SQLAlchemy==1.4.37
-sentry-sdk[flask]==1.0.0
-cachelib==0.1.1
-
-newrelic==6.2.0.156
-notifications-python-client==6.0.2
-python-dotenv==0.17.1
+sentry-sdk[flask]==1.10.1
+cachelib==0.9.0
+newrelic==6.10.0.165
+notifications-python-client==6.4.1
+python-dotenv==0.21.0
 pwnedpasswords==2.0.0
-tldextract==3.1.0
+tldextract==3.4.0
 nanoid==2.0.0
-unidecode==1.2.0
-more-itertools==8.12.0
-
+unidecode==1.3.6
+more-itertools==8.14.0
 # PaaS
 awscli-cwlogs==1.4.6
 aws-embedded-metrics==1.0.7
-
 # Putting upgrade on hold due to new version introducing breaking changes
-Werkzeug==2.0.2
-MarkupSafe==2.0.1
-
+Werkzeug==2.2.2
+MarkupSafe==2.1.1
 # REVIEW: v2 is using sha512 instead of sha1 by default (in v1)
-itsdangerous==2.0.1
-
-git+https://github.com/cds-snc/notifier-utils.git@49.1.0#egg=notifications-utils
-
-# MLWR
-socketio-client==0.5.6
-requests
-requests[security]
-pycryptodome
-
-git+https://bitbucket.org/cse-assemblyline/assemblyline_client.git@v3.7.3#egg=assemblyline_client==v3.7.3
+itsdangerous==2.1.2
+git+https://github.com/cds-snc/notifier-utils.git@50.1.0#egg=notifications-utils
 
-rsa==4.7.2 # not directly required, pinned by Snyk to avoid a vulnerability
+# rsa==4.9  # awscli 1.22.38 depends on rsa<4.8
```

Comment:
This line doesn't need to change, it still ends up pinned to 4.7.2 in requirements.txt on line 108.

Reply:
Hmm @jimleroyer did this one.  Any thoughts @jimleroyer ?

Reply:
You can uncomment it if you want. I was probably testing the dependencies. :)